publishable/views/posts/edit-add.blade.php

```diff
@@ -121,7 +121,10 @@
                         <div class="panel-body">
                             <div class="form-group">
                                 <label for="name">URL slug</label>
-                                <input type="text" class="form-control" name="slug" placeholder="slug" value="@if(isset($dataTypeContent->slug)){{ $dataTypeContent->slug }}@endif">
+                                <input type="text" class="form-control" id="slug" name="slug"
+                                    placeholder="slug"
+                                    {!! isFieldSlugAutoGenerator($dataTypeContent, "slug") !!}
```

Comment:
When modifying BREAD `$dataTypeContent` is defined, but when creating bread it's not defined.
It should handle that since TravisCI fails because of this.